atlp-rwanda/commanders-rn-medica
# Feature: Sign Up and Sing In

#### What does this PR do?
This PR adds the SignUp and SignIn screens with the active state.

#### Description of Task to be completed?
- Created SignUp screen with email and password input fields.
- Added SignIn screen with email and password input fields.
- Implemented active state for the input fields.

#### How should this be manually tested?
1. Run the app.
2. Navigate to the SignUp screen.
3. Verify that the email and password input fields are present.
4. Verify that the input fields change color when focused.
5. Verify that the Sign up button changes color when any input field is active.
6. Navigate to the SignIn screen.
7. Verify that the email and password input fields are present.
8. Verify that the input fields change color when focused.
9. Verify that the Sign in button changes color when any input field is active.

#### Any background context you want to provide?
These changes enhance the user experience by providing visual feedback when the input fields are active.

#### Screenshots (if appropriate)
<div style="display: flex; flex-direction: row;">
  <img src="https://github.com/atlp-rwanda/commanders-rn-medica/assets/114469939/9542043d-eebe-4875-ab06-9d12c39b8649" alt="SignUp Screen" width="200" />
  <img src="https://github.com/atlp-rwanda/commanders-rn-medica/assets/114469939/93ad9a15-bec2-4eb4-b62f-fbb0f3cdaee6" alt="SignIn Screen" width="200" />
  <img src="https://github.com/atlp-rwanda/commanders-rn-medica/assets/114469939/b16ce664-f378-425f-a393-8eb5d054dc71" alt="Active SignUp Screen" width="200" />
</div>

#### Questions:
Any questions or concerns about the changes?
